odbc/src/tests.rs

```diff
@@ -463,3 +465,326 @@ mod get_diag_rec {
         );
     }
 }
+mod env_attributes {
+    use crate::{
+        handles::{Env, EnvState, MongoHandle},
+        tests::UNIMPLEMENTED_FUNC,
+        SQLGetDiagRecW, SQLGetEnvAttrW, SQLSetEnvAttrW,
+    };
+    use odbc_sys::{
+        AttrConnectionPooling, AttrCpMatch, AttrOdbcVersion, EnvironmentAttribute, HEnv,
+        HandleType, Integer, Pointer, SqlReturn,
+    };
+    use std::{ffi::c_void, sync::RwLock};
+
+    const INVALID_ATTR_VALUE: &str = "HY024\0";
+
+    #[test]
+    fn get_set_odbc_version() {
+        let env_handle: *mut _ =
+            &mut MongoHandle::Env(RwLock::new(Env::with_state(EnvState::Allocated)));
+        // SQLSetEnvAttrW(SQL_ATTR_ODBC_VERSION, SQL_OV_ODBC3_80)
```

Comment:
The comments state the obvious, but they at least helped me to quickly get the gist of each function